internal/venice-common/src/main/java/com/linkedin/venice/controllerapi/JobStatusQueryResponse.java

```diff
@@ -57,10 +59,24 @@ public void setStatusDetails(String statusDetails) {
     this.statusDetails = statusDetails;
   }
 
+  /**
+   * N.B.: Older versions of the controller did not support this timestamp, so this can be null.
+   *
+   * @return the UNIX Epoch timestamp when the value of {@link #getStatus()} was last updated (for child controllers),
+   *         null for parent controllers and for older versions of child controllers.
+   */
+  public Long getStatusUpdateTimestamp() {
+    return this.statusUpdateTimestamp;
+  }
+
+  public void setStatusUpdateTimestamp(Long statusUpdateTimestamp) {
```

Comment:
This timestamp is to track version status update?

Reply:
After reading the following code,  I understand this timestamp is used to track the timestamp of the latest status update.